db/migrate/20130703222701_change_pub_hash_to_medium_text.rb

```diff
@@ -1,5 +1,5 @@
 class ChangePubHashToMediumText < ActiveRecord::Migration[4.2]
   def change
-    change_column :publications, :pub_hash, :text, limit: 16_777_215
+    change_column :publications, :pub_hash, :text, limit: 16_777_215 if ActiveRecord::Base.connection.adapter_name.match?(/mysql/i)
```

Comment:
sul_pub's current migrations are coupled to MySQL in a few spots. To make the app able to migrate up and down in both PG and MySQL, use this `if` statement. Once we're on PG in all envs, we can remove these checks for good.

Reply:
i guess the limit for text fields in postgres is at least this big?

Reply:
@peetucket even better, `:text` fields are unlimited in Postgres 🤯

Reply:
quick, download wikipedia and put it in a text field

Reply:
Why are we bothering with this repository business when PG has unlimited text fields?